oss-fuzz/build.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash -eu
+export CLANG_PATH=/usr/local/bin
+export ASAN_SYMBOLIZER_PATH=/usr/local/bin/llvm-symbolizer
+export PACKAGES_PATH="$SRC/hbfa-fl/HBFA:$SRC/edk2"
```

Comment:
There are variables like SRC, OUT, WORK used across the script. Where are those defined?

Reply:
Defined by the OSS-fuzz build process

Reply:
https://google.github.io/oss-fuzz/getting-started/new-project-guide/#buildsh-script-environment